packages/sparql-language-server/src/SparqlLanguageServer.ts

```diff
@@ -79,7 +81,7 @@ export class SparqlLanguageServer extends AbstractLanguageServer<
         // Tell the client that the server works in NONE text document sync mode
         textDocumentSync: this.documents.syncKind[0],
         completionProvider: {
-          triggerCharacters: ['<', ':', '?', '$'],
+          triggerCharacters: ['<', '?', '$'],
```

Comment:
why remove this?

Reply:
Leaving the `:` as a trigger character was actually breaking autocomplete for prefixed names. I.e., if you typed `rdfs` you would get the rdfs completions, but the moment you then hit `:` to continue choosing the local name, the completions would go away (essentially, it was as though the completion state was being reset).

Removing the trigger fixed this; the completion remains even after you type the `:`.

Reply:
cool